### PR TITLE
set accountId and folderId on each message to fix unified inbox replies

### LINKED ISSUE #949: Sometimes sent mail remove subjects
Sometimes when I reply to mails, the subject gets removed for some reason. Can you reproduce @irgendwie @ChristophWurst @PoPoutdoor @zinks-?

### PR DESCRIPTION
**Pull request by ChristophWurst**
Yet another hack to get unified inbox working ;-)

__IMPORTANT:__ clear your local storage with ``localstorage.clear()``. Otherwise you'll run into client side errors when trying to reply to a message that has been cached before. (``accountId`` and ``folderId`` aren't set on old messages).

fixes #949

@jancborchardt @DeepDiver1975 @irgendwie @PoPoutdoor

### CODE REVIEW

**Comment by jancborchardt** [comment]
Can’t test replying from the unified inbox atm because of https://github.com/owncloud/mail/pull/957 ;)

@Gomez @irgendwie @zinks- can you testdrive this?

**Comment by jancborchardt** [comment]
Still happens :(

1. Write an email to your second account from the unified inbox
2. Reply to that mail
3. On the reply, the subject is missing

@DeepDiver1975 can you help here?

File: lib/controller/messagescontroller.php [hunk 1]
```diff
@@ -153,6 +153,10 @@ public function show($accountId, $folderId, $id) {
 			}, $json['attachments']);
 		}
 
+		// Add accountId, folderId for unified inbox replies
+		$json['accountId'] = $accountId;
```
Comment (ChristophWurst) [inline lib/controller/messagescontroller.php]: ``$accountId`` is still set to ``-1`` for the unified inbox. @DeepDiver1975 how do I get the real accountId here? 

Same goes for ``$folderId``.

**Comment by ChristophWurst** [comment]
@jancborchardt should be fixed now. Or at least I cannot reproduce it on my system when using this updated branch.
Again, please delete your local storage before switching, I added new message attributes ;-)

**Comment by Gomez** [comment]
Done the three steps @jancborchardt posted.
Works for me. Subject is there.
:+1:

**Comment by irgendwie** [comment]
LGTM :+1:

**Pull request merged**